houston/queries.go

```diff
@@ -78,6 +78,7 @@ var (
 			version
 			createdAt
 			updatedAt
+			currentTag
```

Comment:
Wait - I just noticed. Isn't this what we ask for in `deployInfo { latest }` already?